pkg/common.go

```diff
@@ -26,6 +25,10 @@ const (
 	// volumes
 	LogsVolume         = "logs"
 	CSISocketDirVolume = "csi-socket-dir"
+
+	// termination settings
+	defaultTerminationMessagePath   = "/dev/termination-log"
```

Comment:
Why do we have `/dev/termination-log` log? Why not `/var/log/termination-log`?

Reply:
Lets use standard path - `/var/log`

Reply:
done